### DISCUSSION TITLE
"Convert labels to SVG" text setting

### DISCUSSION THREAD

**Discussion by OverFloyd** [jgraph/drawio#4137] [Q&A] [answered]
Hello, I just noticed that the "Convert labels to SVG" setting in Export -> SVG... -> Text settings... in gone. Well, as a matter of fact the whole option for Text settings is gone. I use it on a regular basis, may I ask if it will come back anytime soon?

**Comment by alderg** [comment]
Check embed fonts to add an image representation of foreignObjects as a fallback for viewers that do not support foreignObjects. This is the replacement for "Convert labels to SVG" as we plan to deprecate the required infrastructure for converting labels to SVG.

**Comment by OverFloyd** [comment, marked as answer]
I see. Meanwhile I fixed the issue myself by doing what you guys did on the server side, aka exporting as PDF then passing it through Inkscape and that does it for me. I specifically need my text to be converted to SVG as I upload some of my drawings to Wikimedia Commons (which doesn't like text inside SVGs). Thank you.
